applications/FluidDynamicsApplication/tests/test_fluid_auxiliary_utilities.py

```diff
@@ -113,6 +113,39 @@ def testCalculateFlowRateNegativeSkin(self):
         negative_flow_rate_top = KratosFluid.FluidAuxiliaryUtilities.CalculateFlowRateNegativeSkin(top_skin_model_part)
         self.assertAlmostEqual(negative_flow_rate_top, ref_negative_flow_rate_top, 12)
 
+    def testCalculateFluidVolume(self):
```

Comment:
I think there is a copy&paste error in the test name.

Reply:
true